charts/node/templates/statefulset.yaml

```diff
@@ -106,23 +104,19 @@ spec:
           args:
             - -c
             - |
+              set -eu -o pipefail
+              apk add --no-cache wget lz4
```

Comment:
Instead of downloading the wget and lz4, I think it's better to have a generic or dedicated docker image for the init container to have the packages pre-installed. Though the packages are small, downloading the same package when deploying thousand of nodes isn't practical. Also, we might add some packages in the future so it's better to have a docker image for it.

Reply:
Replaced alpine with paritytech/lz4